.gitignore

```diff
@@ -2,6 +2,7 @@
 /.phpunit.result.cache
 /.pint.cache
 /.pint.ci.cache
+/.phpactor.json
```

Comment:
Lets not put tooling specific ignores in here.

You can instead add this to your global gitignore.